modular_skyrat/modules/mapping/code/datums/ruins/space.dm

```diff
@@ -0,0 +1,6 @@
+/datum/map_template/ruin/space/forgottenship
+	name = "CSBC-12"
+	id = "forgottenship"
+	description = "Recently upgraded with popular designs, CSBC-12 has managed to maintain it's incredible track record of not veering off into NT controlled space.. 0% of the time."
```

Comment:
I don't think anyone ever sees this description, but for the sake of it...
Could you just remove the hint of it being 'recently upgraded'? The whole vibe of it is that it's old and it drifted, like Charlie Station, just hasn't been drifting for hundreds of years.

Reply:
Admins see the description when jumping to it through the debug menu, usually used for flavor text and background lore. I'll change it up, sure! Donk pockets are already there though, I used the same random spawner most maps and base cybersun uses.